src/test/java/hw1/data_providers/ArithmeticDataProviders.java

```diff
@@ -0,0 +1,66 @@
+package hw1.data_providers;
+
+import org.testng.annotations.DataProvider;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ArithmeticDataProviders {
```

Comment:
ThreadLocalRandom.current() should be extracted to class field

Reply:
Thanks. Don't think about it. Fixed